R/scale-manual.r

```diff
@@ -132,8 +132,9 @@ manual_scale <- function(aesthetic, values = NULL, breaks = waiver(), ..., limit
     force(values)
   }
 
-  if (is.null(limits)) {
-    limits <- names(values)
+  if (is.null(limits) && !is.null(names(values))) {
+    force(values)
```

Comment:
Why is this line needed? Wasn't `values` forced in the last `if` section?

Reply:
You're right, it must have slipped my attention. Apologies!

Reply:
I see, thanks.